v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -56,6 +56,16 @@ export const MovableLine = (props: {
     xSubAxesCount = layout.getAxisMultiScale('bottom')?.repetitions ?? 1,
     ySubAxesCount = layout.getAxisMultiScale('left')?.repetitions ?? 1
 
+  const equationContainerClass = useCallback(() => {
+    const classFromKey = model.setClassNameFromKey(lineKey)
+    return `movable-line-equation-container${lineKey && lineKey !== '' ? `-${classFromKey}` : ''}`
+  }, [lineKey, model])
+
+  const equationContainerSelector = useCallback(() => {
+    const gridContainerClass = `graph-adornments-grid.${instanceId}`
+    return `.${gridContainerClass} .${equationContainerClass()}`
+  }, [instanceId, equationContainerClass])
```

Comment:
Minor point, but since these don't rely on many values from the surrounding component, my inclination would be to define them globally outside the component so you don't have to use `useCallback`. It could look something like:
```typescript
function equationContainer(model: IMovableLineModel, instanceId: string, lineKey?: string) {
   ... string manipulation code ...
   return { equationContainerClass, equationContainerSelector }
}
```
and then the calling code would just be:
```typescript
  const { equationContainerClass, equationContainerSelector } = equationContainer(model, instanceId, lineKey)
```